Content.Server/GameTicking/GameTicker.RoundFlow.cs

```diff
@@ -414,7 +414,8 @@ private async void SendRoundEndDiscordMessage()
 
                 content = Loc.GetString("discord-round-notifications-end-ping", ("roleId", DiscordRoundEndRole));
                 payload = new WebhookPayload { Content = content };
-                payload.AllowedMentions.AllowRoleMentions();
+                payload.AllowedMentions = new();
+                payload.AllowedMentions.Value.AllowRoleMentions();
```

Comment:
With the other review this isn't necessary anymore.
```suggestion
                payload.AllowedMentions.AllowRoleMentions();
```